code/scripting/api/objs/debriefing.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "mission/missionbriefcommon.h"
+#include "missionui/missiondebrief.h"
+#include "scripting/ade_api.h"
+
+namespace scripting {
+namespace api {
+
+DECLARE_ADE_OBJ(l_DebriefStage, debrief_stage);
+
+DECLARE_ADE_OBJ(l_Debrief, debriefing);
```

Comment:
Similar to the other SCPUI PR's, this is a problem: We don't want to copy the debrief object here